roles/ceph-mon/tasks/docker/fetch_configs.yml

```diff
@@ -21,7 +21,7 @@
 
 - name: try to fetch ceph config and keys
   copy:
-    src: "{{ playbook_dir }}/{{ fetch_directory }}/docker_mon_files/{{ item.0 }}"
+    src: "{{ fetch_directory }}/docker_mon_files/{{ item.0 }}"
```

Comment:
Ok, that's good to know. I still think having ``playbook_dir`` here is incorrect though, after https://github.com/ansible/ansible/issues/18090 it won't be an issue and until then we have a workaround.